### ISSUE TITLE
Example : generate for GO

### ISSUE THREAD

**Issue by fiftoine** [google/protobuf-gradle-plugin#263]
Hi,

From a set of proto files, I would like to generate stubs for both java and go.

Reading the doc I can't find a way of generating stubs for go even if it's mentioned in others issues.

Could you provide an example on how to generate for go alone and for java and go?

Thanks in advance

**Comment by zhangkun83** [comment]
You should follow the Step 2 from the [protobuf documentation](https://developers.google.com/protocol-buffers/docs/gotutorial#compiling-your-protocol-buffers) to install the Go protocol buffers plugin.

Then add `go {}` to the `plugins` of all protobuf generating tasks:
```gradle
protobuf {
  generateProtoTasks {
    all().each { task ->
      task.plugins {
        go { }
      }
    }
  }
}
```

Run `./gradlew generateProto` and you should find the generated files under `build/generated/source/proto`. We have done nothing to integrate the generated files into Go compilation, you would have to wire them up manually.

**Comment by zhangkun83** [comment]
@Groostav, to use the C# grpc plugin, you would need to first define it, with either an artifact spec or a local path, e.g.,
```gradle
protobuf {
  ...
  // Configure the codegen plugins
  plugins {
    // Define a plugin with name 'grpc'
    grpc {
      path = file("$projectDir/../.kinda-nuget/packages/grpc.tools/1.15.0/tools/windows_x64/grpc_csharp_plugin.exe").absolutePath
    }
    // Any other plugins
    ...
  }
  ...
}
```

**Comment by zhangkun83** [comment]
OP's problem could be solved by allowing referring "undefined" plugins, since `protoc` will search in `$PATH` for the plugin if its path is not specified. I have created #270 for that.

Unfortunately the CI is broken and we are short-staffed, and we can't make a release with CI broken. Any help to fix the CI is welcome.

**Comment by zhangkun83** [comment]
0.8.7 is released and includes #270
